pinot-controller/src/test/java/org/apache/pinot/controller/helix/PinotControllerModeTest.java

```diff
@@ -67,6 +72,25 @@ public void testDualModeController()
         "Failed to start " + config.getControllerMode() + " controller in " + TIMEOUT_IN_MS + "ms.");
     Assert.assertEquals(_controllerStarter.getControllerMode(), ControllerConf.ControllerMode.DUAL);
 
+    // Enable the lead controller resource.
```

Comment:
You also need a method to wipe zk before each test case

Reply:
Check the test code. There's a method called `protected void startController(ControllerConf config, boolean deleteCluster)`. If  `deleteCluster` is true, it'll wipe out the existing cluster info. So no need to implement a wipe out method any more.